src/RedisEnterpriseCache/readme.md

```diff
@@ -0,0 +1,132 @@
+<!-- region Generated -->
+# Az.RedisEnterpriseCache
+This directory contains the PowerShell module for the RedisEnterpriseCache service.
+
+---
+## Status
+[![Az.RedisEnterpriseCache](https://img.shields.io/powershellgallery/v/Az.RedisEnterpriseCache.svg?style=flat-square&label=Az.RedisEnterpriseCache "Az.RedisEnterpriseCache")](https://www.powershellgallery.com/packages/Az.RedisEnterpriseCache/)
+
+## Info
+- Modifiable: yes
+- Generated: all
+- Committed: yes
+- Packaged: yes
+
+---
+## Detail
+This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 1.8.1 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.RedisEnterpriseCache`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+## Run Generation
+In this directory, run AutoRest:
+> `autorest`
+
+---
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+require:
+  - $(this-folder)/../readme.azure.noprofile.md
+  - $(repo)/specification/redisenterprise/resource-manager/readme.md
```

Comment:
Instead of using branch, please try to lock the swagger you use through commit ID. Please see https://github.com/Azure/azure-powershell/blob/generation/src/Databricks/readme.md for an example.